README.md

```diff
@@ -101,6 +101,21 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one.
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
+4. Create a fork off of OfficeDev/Office-Addin-Scripts in GitHub
+5. In a command prompt:
```

Comment:
Should specify what folder to CD to before running these commands.

Reply:
@igor-ribeiiro Ricky makes a good point. For step 5 (in a command prompt), please add a first step: `1. cd Office-Addin-Scripts`

Also, a step 8 could be added. 8. Go to https://github.com/OfficeDev/Office-Addin-Scripts and create a pull request.